articles/reliability/reliability-fabric.md

```diff
@@ -39,8 +39,9 @@ Fabric makes commercially reasonable efforts to support zone-redundant availabil
 
 Fabric makes commercially reasonable efforts to provide availability zone support in various regions as follows:
 
-| **Americas**         | **Power BI**                                                       | **Datamarts**                                                      | **Data Warehouses**                                                | **Real-Time Analytics**                                            | **Data Factory (pipelines)**                                       | **Data Engineering**                                               |
+|                      | **Power BI**                                                       | **Datamarts**                                                      | **Data Warehouses**                                                | **Real-Time Analytics**                                            | **Data Factory (pipelines)**                                       | **Data Engineering**                                               |
```

Comment:
```suggestion
| **Region**           | **Power BI**                                                       | **Datamarts**                                                      | **Data Warehouses**                                                | **Real-Time Analytics**                                            | **Data Factory (pipelines)**                                       | **Data Engineering**                                               |
```
adding header to comply with validation rules

Reply:
thanks for the fix Jak